.gitignore

```diff
@@ -186,7 +186,12 @@ copier_scripts/
 /cms/static_compiled
 
 node_modules/
-/static/
+# Ignore the top-level "static" directory
+/static
+
+# Don't ignore "static/" within subdirectories
+!/*/static/
```

Comment:
I think this may be redundant? We already have folders names `static` which are not ignored, as the first rule is only applied to the root because of the leading slash.

Reply:
Revert, can't remember why I changed this. [`19cd39d` (#163)](https://github.com/ONSdigital/dis-wagtail/pull/163/commits/19cd39de350fcc17519fe05f42b38c54af137d7a)